docs/platform/howto/byoc/create-custom-cloud.md

```diff
@@ -727,6 +727,27 @@ Your new custom cloud is ready to use only after its status changes to
      --display-name "CUSTOM_CLOUD_DISPLAY_NAME"
    ```
 
+   Replace the following:
+
+   - `ORGANIZATION_IDENTIFIER` with the identifier of your Aiven organization that is to be
+     connected with your own cloud account to create the custom cloud,
+     for example `org484338bebd2`. You can extract `ORGANIZATION_IDENTIFIER` from the
+     output of the `avn organization list` command.
+   - `DEPLOYMENT_MODEL_NAME` with the type of [network architecture](/docs/platform/concepts/byoc#byoc-deployment)
+     your custom cloud uses:
+     - `standard_public` (public) model allows the Aiven control plane to connect to the
+       service nodes via the public Internet.
+     - `standard` (private) model routes traffic through a proxy for additional security
+       utilizing a bastion host physically separated from the Aiven services.
+   - `CLOUD_PROVIDER_NAME` with the name of a cloud provider to be used, which is `google`.
+   - `CLOUD_REGION_NAME` with the name of a Google region where to create your custom cloud,
+     for example `europe-north1`.
```

Comment:
I think it could be useful for a user to link to a list of available options. They are the ones listed in https://aiven.io/docs/platform/reference/list_of_clouds, but without the `google-` prefix. (there's probably google's own list of regions too, but we don't necessarily take new ones into use immediately, so our own docs would be safer).